### PR TITLE
A fix and some cleanup

### PR DESCRIPTION
**Pull request by jenswi-linaro**

### CODE REVIEW

File: host/xtest/regression_4000.c [hunk 1]
```diff
@@ -5299,6 +5299,7 @@ static void xtest_tee_test_4012(ADBG_Case_t *c)
 }
 ADBG_CASE_DEFINE(regression, 4012, xtest_tee_test_4012,
 		"Test seeding RNG entropy");
+#endif /*CFG_SYSTEM_PTA*/
 
 static void xtest_tee_test_4013(ADBG_Case_t *c)
```
Comment (jforissier) [inline host/xtest/regression_4000.c:5304]: This one uses `TA_CRYPT_CMD_DERIVE_TA_UNIQUE_KEY{,SHM}` which depend on `CFG_SYSTEM_PTA`

**Review by jforissier — COMMENTED** [review]
For commit "regression 4013-4016: remove CFG_SYSTEM_PTA dependency": one comment below.

But perhaps it would be better to remove the `CFG_SYSTEM_PTA` conditionals altogether? That is deal gracefully with a possible open session error in test 4012.

**Comment by jforissier** [comment]
For "regression 4016: drop _ed25519 suffix" and "regression 4016: fix TEE_ATTR_EDDSA_PREHASH parameter":

`Reviewed-by: Jerome Forissier <jerome.forissier@linaro.org>`

**Comment by jenswi-linaro** [comment]
I'm dropping "regression 4013-4016: remove CFG_SYSTEM_PTA dependency". I'll open a new PR for that one.

Tags applied to the other commits.

**Pull request merged**